documentation/manual-lobster_cpptest.md

```diff
@@ -0,0 +1,115 @@
+# Tracing to C/C++ test requirements
+
+## Setup and requirements
+
+You will need a C/C++ file or a directory containing.
+
+C/C++ files should have one of these extensions to be evaluated by this tool: `.cpp, .cc, .c or .h`
+
+## Preparing C/C++ test comments with requirements
+The test functions should be specified with one the following names: 
+        `TEST`,
+        `TEST_P`,
+        `TEST_F`,
+        `TYPED_TEST`,
+        `TYPED_TEST_P`,
+        `TYPED_TEST_SUITE`,
+        `TEST_P_INSTANCE`,
+        `TEST_F_INSTANCE`
+
+In your test you need to add comments. For example:
+
+```C++
+/**
+ * @requirement CB-#0815, CB-#0816,
+ * 				CB-#0817
+ * @requirement CB-#0818 CB-#0819
+ * 				CB-#0820
+ */
+TEST(ImplicationTest, BasicTest) {}
+```
+
+* Each test can have multiple test methods defined in the comment part.
+
+* These can be simply defined with `/** Test methods */` or `/// Test methods` format. 
+
+* Test methods can be separated by commas or spaces.
+
+* Note: The `markers` should be exactly close to the test functions (without any empty lines)
+
+
+```C++
+/**
+ * @requirement CB-#0815, CB-#0816
+ */
+TEST(ImplicationTest, BasicTest) {}
+```
+
+```C++
+///
+/// @requiredby FOO0::BAR0
+///
+TEST(ImplicationTest, BasicTest) {}
+```
+The used regex for each test-mehods are like follows:
+
+@requirement
+: ```r"(CB-#\d+)"```
+: ```r"({provided_codebeamer_link in config-file}(?P<number>\d+))"```
+
+@requiredby
+: ```r"(\w*::\w+)"```
+
+@defect
+: ```r"(CB-#\d+)|(OCT-#\d+)"```
+
+
+## Preparing cpptest config-file
+
+The config-file determines which `markers` should be extracted in which output-files also you can 
+specify the used `kind` in each output-files. 
+
+In addition, you have to provide the `codebeamer-url`:
+
+```cpp.config
+{
+	"output": {
+		"unit_tests.lobster" : 
+            {
+                "markers": ["@requirement"],
+                "kind": "req"
+            },
+        "components_tests.lobster" :
+            {
+                "markers": ["@requiredby", "@requirement"],
+                "kind": "imp"
+            }
+	},
+	"codebeamer_url": "https://codebeamer.example.com/test"
+}
+ ```
+
+## Creating lobster files
+
+Run the `lobster_cpptest` tool, pointing it to one or more C/C++ files, or a directory containing one or more C/C++ files. 
+
+For example `lobster_cpptest .` should find all your C/C++ files in the root directory.
```

Comment:
better say "current directory" instead of "root directory". "root" is something else in linux, see [root](https://hpc.nmsu.edu/onboarding/linux/files-folders/#:~:text=The%20directory%20at%20the%20highest,directory%2C%20lies%20beneath%20another%20directory.)